### PR TITLE
BRIDGE-1164 Show "learn more" in the consent quiz failed step

### PR DESCRIPTION
**Pull request by syoung-smallwisdom**
This repurposes the APHWebViewStepViewController which is used to display the monthly report to allow this view controller to be used to load htmlContent.

### CODE REVIEW

File: mPowerSDK/Dashboard/APHWebViewStepViewController.m [hunk 1]
```diff
@@ -48,7 +54,40 @@ @interface APHWebviewViewController () <UIWebViewDelegate>
 
 @end
 
-@implementation APHWebviewViewController
+@implementation APHWebViewStepViewController
+
++ (instancetype)new {
```
Comment (Erin-Mounts) [inline mPowerSDK/Dashboard/APHWebViewStepViewController.m]: Not keen on the idea of overriding new like this. General expectation is that [X new] is equivalent to [[X alloc] init]. Can we use a different name?

File: mPowerSDK/Dashboard/APHWebViewStepViewController.m [hunk 2]
```diff
@@ -57,13 +96,21 @@ - (void)viewDidLoad {
...
+    if (self.displayURLString != nil) {
+        NSURLRequest *request = [[NSURLRequest alloc] initWithURL:[NSURL URLWithString:self.displayURLString]];
+        [self.webview loadRequest:request];
+    }
+    else {
+        [self.webview loadHTMLString:self.htmlContent baseURL:[NSURL URLWithString:@"http://content"]];
+    }
+    
+    if (!self.shouldShowShareOptions) {
+        self.toolbarBottomConstraint.constant = -1*self.toolbar.bounds.size.height;
```
Comment (Erin-Mounts) [inline mPowerSDK/Dashboard/APHWebViewStepViewController.m]: Readability preference: spaces around operators means I have to squint less ;)

File: mPowerSDK/Onboarding/APHOnboardingManager.m [hunk 3]
```diff
@@ -328,29 +329,26 @@ - (void)taskViewController:(ORKTaskViewController *)taskViewController stepViewC
...
+- (BOOL)taskViewController:(ORKTaskViewController *)taskViewController hasLearnMoreForStep:(ORKStep *)step {
+    return [self usesLearnMoreViewControllerForStep:step];
+}
+
+- (void)taskViewController:(ORKTaskViewController *)taskViewController learnMoreForStep:(ORKStepViewController *)stepViewController {
+    ORKStep *step = stepViewController.step;
+    if (![self usesLearnMoreViewControllerForStep:step]) {
+        return;
+    }
+    NSString *htmlContent = ((SBADirectNavigationStep*)step).learnMoreHTMLContent;
```
Comment (Erin-Mounts) [inline mPowerSDK/Onboarding/APHOnboardingManager.m]: Nit: space before *

**Comment by Erin-Mounts** [comment]
A few nits, otherwise looks good

**Comment by syoung-smallwisdom** [comment]
@Erin-Mounts Fixed the nits. WRT using `new` I agree and forgot that back when I started the project for Fluke I'd cringed at that pattern (Sorry).

**Comment by Erin-Mounts** [comment]
looks good

**Pull request merged**